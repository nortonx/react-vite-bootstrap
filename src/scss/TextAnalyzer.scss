.text-analyzer {
  height: 72vh;
}