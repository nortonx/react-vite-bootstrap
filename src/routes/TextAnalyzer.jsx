import "../scss/TextAnalyzer.scss";
import Analyzer from "../components/text-analyzer/Analyzer";
const TextAnalyzer = () => {
  return(
    <div className="container text-analyzer" data-testid="text-analyzer">
      <Analyzer />
    </div>
  );
};

export default TextAnalyzer;