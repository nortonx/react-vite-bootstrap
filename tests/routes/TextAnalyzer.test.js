import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";

import TextAnalyzer from "../../src/routes/TextAnalyzer";

describe("TextAnalyzer route", () => {
  it("Should render the component and match snapshot", async () => {
    render(<TextAnalyzer />);
    const element = await screen.findByTestId("text-analyzer");
    expect(element).toBeInTheDocument();
    const textAnalyzer = renderer.create(<TextAnalyzer />);
    expect(textAnalyzer).toMatchSnapshot();
  })
})