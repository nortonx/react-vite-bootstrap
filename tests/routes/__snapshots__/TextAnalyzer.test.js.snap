// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAnalyzer route Should render the component and match snapshot 1`] = `
<div
  className="container text-analyzer"
  data-testid="text-analyzer"
>
  Text analyzer content
</div>
`;
